masterlist.yaml

```diff
@@ -810,6 +810,10 @@ plugins:
       - Relev
       - Scripts
       - Stats
+    inc: 
+    - name: 'EVE FNV - ALL DLC.esp'
+      condition: 'active("EVE FNV - ALL DLC.esp") and not file("Project Nevada - Equipment - EVE ALL DLC Patch.esp")'
+      msg: 'Incompatible with "EVE FNV - ALL DLC.esp", a patch can be found here: [ "https://github.com/gutfuckllc/Patchy-s-Patchonarium-for-NV/releases/download/0.1/Project.Nevada.-.Equipment.-.EVE.ALL.DLC.Patch.zip" ]'
```

Comment:
Which alternative should I use?

Reply:
I'd add a conditional message about it, see how `patch3rdParty` is used in the skyrimse masterlist. However, linking to a zip file hosted on your GitHub is not ideal, because:

- There's no indication of safety - for all users know it's a zip file stuffed with viruses. It may not be now, but your account may be compromised.
- There's no indication of quality - without a user rating, review or comment system, users can't tell if what they're downloading is legit. Again, it might be now, but it's an ongoing trust issue.
- The user isn't able to see what they're about to download before downloading it (because you link directly to the zip).
- It's prone to breaking, e.g. if you delete your GitHub account, or that repository, or that release, and it's relatively difficult to find someone to transfer ownership to, and have URLs redirect correctly.
- It's not very discoverable. Probably the only way anyone will ever find your patches is through the link LOOT provides, which I feel puts LOOT in a slightly dubious position as a kind of gatekeeper.

It would be much better if you uploaded the patches to somewhere like the Nexus, which has features to mitigate the above risks. If you don't want to do that, I'm not willing to link to the patches.